Deeplake docs fixes

adolkhan opened the pull request:
Updated Deep Lake Vector Store's examples to use latest up-to-date API also fixed deprecated embedding_function related warning

adolkhan committed:
Jupyter notebook changes

adolkhan committed:
removed uneccessary folder

baskaryan commented:
you'll want to add redirects to `docs/docs_skeleton/vercel.json` for the two notebooks that were renamed, e.g. 
```json
{
  "source": "/docs/integrations/vectorstores/deeplake"
  "destination": "/docs/integrations/vectorstores/activeloop_deeplake"
}
```

adolkhan committed:
Docs fixes

adolkhan committed:
poetry.lock file

adolkhan commented:
Thanks for reviewing! Addressed comments. Hopefully I haven't messed up anything! Also what do you think of code understanding example, should move two examples into one place, as they basically duplication of each other?

baskaryan commented:
fixed up and landed in #9275, thanks @adolkhan!
